test_runner/src/main/kotlin/ftl/environment/ListLocales.kt

```diff
@@ -33,9 +33,9 @@ fun List<Locale>.getLocaleDescription(localeId: String) =
 
 private fun List<Locale>.findLocales(localeId: String) = find { it.id == localeId }
 
-private fun Locale.prepareDescription() = """
-    id: $id
-    name: $name
+fun Locale.prepareDescription() = """
+id: $id
+name: $name
 """.trimIndent().addRegionIfExist(region).addTagsIfExists(this)
```

Comment:
it should be moved to `presentation` package

Reply:
moved.